app/controllers/documents_controller.rb

```diff
@@ -6,19 +6,26 @@ class DocumentsController < ApplicationController
   include ActionView::Helpers::TextHelper
 
   before_action :fetch_document, only: [:edit, :show, :publish, :update]
-  before_action :permitted?, if: :document_type
+  before_action :check_authorisation, if: :document_type
 
-  def index
-    page = filtered_page_param(params[:page])
-    per_page = filtered_per_page_param(params[:per_page])
+  after_action :verify_authorized
```

Comment:
Ok, but then could we then do this on a subsequent PR? Having this enforcement and having a mechanism which catches forgotten authorisation is one of the biggest benefits of a framework like Pundit.